File: Fundamentos de Desenvolvimento com C#/AT/EX08/Gerente.cs
namespace EX08 {
    internal class Gerente : Funcionario {
        public Gerente(string nome, string cargo, double salarioBase)
            : base(nome, cargo, salarioBase) {
        }

        public override double CalcularSalario() {
            return SalarioBase * 1.20; 
        }
    }
}

File: Fundamentos de Desenvolvimento com Java/AT/src/EX08/Gerente.java
package EX08;

public class Gerente extends Funcionario {

    public Gerente(String nome, double salarioBase) {
        super(nome, salarioBase);
    }

    @Override
    public double calcularSalario() {
        return salarioBase + (salarioBase * 0.20);
    }
}
